test/sharness/t0020-init.sh

```diff
@@ -58,11 +58,13 @@ test_expect_success "ipfs config succeeds" '
 	test_cmp expected_config actual_config
 '
 
+test_check_peerid() {
```

Comment:
Ah ok.

But as the output of `ipfs "$@" id -f="<id>"` is already tested with:

```
	ipfs "$@" id -f="<id>" >actual &&
 	test_cmp expected actual
```

I think it is better to just use it like this:

```
test_expect_success "config setup" '
	api_fromcfg=$(ipfs config Addresses.API) &&
	peerid=$(ipfs config Identity.PeerID) &&
	test_check_peerid "$peerid"
'
```

I plan to do something like that when this PR and PR #1626 are merged.